librz/core/cmd/cmd_search.c

```diff
@@ -529,3338 +153,363 @@ static int _cb_hit(RzSearchKeyword *kw, void *user, ut64 addr) {
 	return true;
 }
 
-static int c = 0;
+static int progress = 0;
 
 static inline void print_search_progress(ut64 at, ut64 to, int n, struct search_parameters *param) {
-	if ((++c % 64) || (param->outmode == RZ_MODE_JSON)) {
+	if ((++progress % 64) || (param->outmode == RZ_MODE_JSON)) {
 		return;
 	}
 	if (rz_cons_singleton()->columns < 50) {
-		eprintf("\r[  ]  0x%08" PFMT64x "  hits = %d   \r%s",
-			at, n, (c % 2) ? "[ #]" : "[# ]");
+		rz_cons_printf("\r[  ]  0x%08" PFMT64x "  hits = %d   \r%s",
```

Comment:
i'd use RZ_LOG_INFO because this is not the real output of the command, it's just some update on the status.